rest-api/dao/participant_counts_over_time_service.py

```diff
@@ -181,7 +181,7 @@ def get_enrollment_status_sql(self, filters_sql_ps, filters_sql_p):
             FROM calendar c2
             LEFT OUTER JOIN
             (SELECT COUNT(*) cnt,
-                (CASE WHEN enrollment_status = 1 THEN
+                (CASE WHEN (enrollment_status = 1 OR enrollment_status IS NULL) THEN
```

Comment:
As discussed offline, this needs to be updated to use table_prefix:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/participant_counts_over_time_service.py#L124